package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@creately/rxdata",
-  "version": "5.1.3",
+  "version": "5.1.4",
```

Comment:
package-lock.json should be updated as well.

Reply:
Updated